apps/condo/domains/organization/access/Organization.js

```diff
@@ -76,18 +76,23 @@ async function canReadOrganizations ({ authentication: { item: user }, listKey }
     return { OR: accessConditions }
 }
 
-async function canManageOrganizations ({ authentication: { item: user }, operation }) {
+async function canManageOrganizations ({ authentication: { item: user }, operation, listKey, originalInput }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
-    if (user.isSupport || user.isAdmin) return true
+    if (user.isAdmin || user.isSupport) return true
 
-    if (operation === 'create') {
-        return false
-    } else if (operation === 'update') {
-        // user is inside employee list and is not blocked
-        return {
-            employees_some: { user: { id: user.id }, role: { canManageOrganization: true }, isBlocked: false, deletedAt: null },
-        }
+    // You should use "registerNewOrganization"
+    if (operation === 'create') return false
```

Comment:
I guess it's should be true for admin and supports as well...

Reply:
A complete access update involves some refactoring, let's create a separate task for this?

Reply:
I don’t want a small task about adding a new field to turn into a huge task with refactoring :)

Reply:
Now there is a task DOMA-7240 for this